ubuntu-classic-2504-amd64.json

```diff
@@ -20,15 +20,29 @@
         {
             "name": "pc-kernel",
             "type": "kernel",
-            "default-channel": "25.04/stable",
-            "id": "pYVQrBcKmBa0mZ4CCN7ExT6jH8rY1hza"
+            "default-channel": "24/edge/nvidia-components-dev",
+            "id": "pYVQrBcKmBa0mZ4CCN7ExT6jH8rY1hza",
+            "components": {
+                "nvidia-550-ko": {
+                    "presence": "optional"
+                },
+                "nvidia-550-user": {
+                    "presence": "optional"
+                }
```

Comment:
What about nouveau? Is it not being offered as an option?

Reply:
I think it's best to omit nouveau for now.  There is no plan to address that case for 25.04, so it would be a waste of space for the moment.  I suspect that handling that needs some matching UX in the bootstrap UI, and if that were in place we could do the appropriate driver mapping and install nouveau.

Reply:
Ok, thanks for the explanation.